### ISSUE TITLE
A way to hide from UI completely?

### ISSUE THREAD

**Issue by dfischer** [JackAdams/constellation#10]
When launching this in Cordova it takes up a lot of room and there doesn't seem to be a way to remove this from the UI completely (hide it).

Any idea?

**Comment by JackAdams** [comment]
From a mobile app being tested on the phone itself (or a simulator), yes, you're right that there's no way to remove it completely from the UI if the `constellation:position` package is present (and it _is_ present when using the `babrahams:constellation` distro).  The reason for this is that there would be no way to open and close the UI otherwise (the `constellation:position` package gives the the little dot to open it and lets you close it down to the little dot again by clicking the `Position` tab).

If you're using the mobile view in Chrome, you can use the `Config` tab to remove the `Position` tab and use Ctrl + C to toggle the UI.

**Comment by dfischer** [comment]
Yeah. I'd say it would probably make sense to add a option in config to temporarily hide the whole UK until you refresh to get around this.

**Comment by JackAdams** [comment]
Are we talking about emulation/devices or just the mobile view on Chrome?  Because on Chrome, you can use the config tab to hide the position tab and just use Crtl + C to completely hide the UI.  In emulators/devices, Ctrl + C isn't going to work, so we need some sort of visual presence to click.

How do you refresh in a device/emulator -- is there an easy way of doing that?  If so, that might be worth doing.  It would be easy enough.

**Comment by dfischer** [comment]
Probably relaunching the app is enough to cause the refresh. Yeah I'm speaking about emulator, not so much chrome.
